build/commands/lib/start.js

```diff
@@ -84,6 +84,15 @@ const start = (passthroughArgs, buildConfig = config.defaultBuildConfig, options
     killSignal: 'SIGTERM'
   }
 
+  if (process.platform === 'darwin' && !config.isReleaseBuild()) {
```

Comment:
let's move it before `let cmdOptions = {` block. Seems like all `braveArgs` manipulation ends before it.

Reply:
moved.